morgan/roster.json

```diff
@@ -41,6 +41,9 @@
     {
       "name": "Muhammed Ismael Ali",
       "github": "Ismael76"
+    },
+    {
+      "name": "Vincent Okolo",
+      "Github": "VAOkolo"	
```

Comment:
Can you please write github in lowercase? you are also missing the closing square bracket of the students array :)

Reply:
Will adjust and give it another go. Thanks mate :)

Reply:
should be ready to merge in